packages/runtime/client-api/src/api/codeLoader.ts

```diff
@@ -70,6 +71,7 @@ export class Chaincode implements IFluidDataStoreFactory {
             sequence.SharedObjectSequence.getFactory(),
             sequence.SharedNumberSequence.getFactory(),
             ConsensusQueue.getFactory(),
+            ConsensusRegisterCollection.getFactory(),
```

Comment:
is this part of the agent scheduler change?

Reply:
Yes - the replay-tool uses client-api, and the replay-tool is used for the snapshot tests.  Since the snapshots include `AgentScheduler` (which uses `ConsensusRegisterCollection`) they need it in the registry -- previously it would be implicitly included by `ContainerRuntimeDataStoreRegistry`, now it needs to be explicit.